### PR TITLE
pg803's Card

### PR DESCRIPTION
**Pull request by pg803**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ pg803's card START ________  -->
+        <div class="card">
+          <p class="name">pg803</p>
```
Comment (dilshankarunarathne) [inline index.html:119]: Changing this line to `<i class="fab fa-github"></i>` will correct the icon.

**Review by dilshankarunarathne — APPROVED** [review]
Hello @pg803, your code looks okay. 
I changed the Twitter icon into GitHub icon.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @pg803, please make these changes, so we will be able to review and merge your pull request.

File: index.html [hunk 2]
```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ pg803's card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:117]: Please make sure to leave a blank line between the template and your code as well.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]

**Comment by pg803** [comment]
Hi,
I have made the changes,
I had thought I had changed it.

**Comment by dilshankarunarathne** [comment]
Congratulations 🎉❤
Your pull request was merged. 
You will be able to see your card on the [project website](https://syknapse.github.io/Contribute-To-This-Project/).

**Pull request merged**